packages/core/src/part.js

```diff
@@ -357,20 +357,68 @@ Part.prototype.generateTransform = function(transforms) {
   }
 }
 
-/** Chainable way to set the grain property */
-Part.prototype.setGrain = function (grain = 90) {
-  this.attributes.set('data-grain', grain)
+/** Chainable way to add the cut info */
+Part.prototype.addCut = function (cut=2, material='fabric', identical=false) {
+  if (cut === false) {
+    if (this.cut.materials[material]) delete this.cut.materials[material]
+    else this.context.raise.error(`Tried to remove a material that is not set`)
+    return this
+  }
+  if (typeof this.cut.materials[material] === 'undefined') this.cut.materials[material] = {}
+  if (typeof cut !== 'number') {
+    this.context.raise.error(`Tried to set cut to a value that is not a number`)
+    return this
+  }
+  if (typeof material !== 'string') {
+    this.context.raise.error(`Tried to set material to a value that is not a string`)
+    return this
+  }
+  this.cut.materials[material].cut = cut
+  this.cut.materials[material].identical = identical
+
+  return this
+}
+
+/** Chainable way to remove (some) cut info */
+Part.prototype.removeCut = function (material=false) {
+  if (!material) {
+    this.context.raise.warning('Called part.removeCut() without any parameters. Not removing anything')
+    return this
+  }
+  if (typeof material !== 'string') {
+    this.context.raise.error(`Tried to set material to a value that is not a string`)
+    return this
+  }
+  if (this.cut.materials[material]) delete this.cut.materials[material]
+  else this.context.raise.error(`Tried to remove a material that is not set`)
 
   return this
 }
 
-/** Chainable way to set the grain property */
-Part.prototype.setCut = function (cut = { count: 2, mirror: true, onFold: false }) {
-  this.attributes.set('data-cut', cut)
+/** Chainable way to add the grain info */
+Part.prototype.setGrain = function (grain=false) {
+  if (grain === false) {
+    this.context.raise.warning('Called part.setGrain() without any parameters. Not changing anything')
+    return this
+  }
```

Comment:
I actually think it should remove the grain. I guess a part that doesn't care about grain is pretty rare, but maybe it should be an option?

Reply:
A part that is a circle of fabric? For example, when making circular/round fabric buttons.

Reply:
Also, not familiar with Javascript or how grain works, but will trying to set a grain of 0 get caught by this? I'm assuming grain is an angle and 0 is a valid value.

Reply:
In js, `===` is exact equality, so this will only catch false values, not falsy

Reply:
Not sure what to make of *remove the grain* so I've set it to `false`. I think we should always have `grain` as a property, rather than remove it from the object.
Setting it to `false` is explicit, so went with that.